filament/src/materials/blit.mat

```diff
@@ -0,0 +1,30 @@
+material {
+    name : blit,
+    parameters : [
+        {
+            type : sampler2d,
+            name : color,
+            precision: high
```

Comment:
Why is the sampler precision set to high? We use at most RGBA16F which fits in `mediump`. UVs have to use `highp`, but not the samplers themselves.

Reply:
good catch